fastlane/Fastfile

```diff
@@ -155,8 +155,8 @@ platform :ios do
 
   desc "Prepare next version"
   lane :prepare_next_version do |options|
-    current_version_number = current_version_number
-    major, minor, _ = current_version_number.split('.')
+    old_version_number = current_version_number
+    major, minor, _ = old_version_number.split('.')
```

Comment:
this was broken: since the variable and the method had the same name, `split` was actually getting called on the method.

Reply:
it was broken on #514